packages/core/components/HeaderRibbon/HelpControl.tsx

```diff
@@ -0,0 +1,248 @@
+                items: [
+                    {
+                        key: "Annotation Hierarchy",
+                        text: "Annotation Hierarchy",
+                        title:
+                            "How to organize the files in the file list into hierarchical folders using the annotation hierarchy",
+                        onClick: () => {
+                            props.onCollapse();
+                            dispatch(selection.actions.selectTutorial(ORGANIZE_FILES_TUTORIAL));
+                        },
+                        itemProps: OPTION_PROPS,
+                    },
+                    {
+                        key: "Filtering",
+                        text: "Filtering",
+                        title: "How to filter files in the file list",
+                        onClick: () => {
+                            props.onCollapse();
+                            dispatch(selection.actions.selectTutorial(FILTER_FILES_TUTORIAL));
+                        },
+                        itemProps: OPTION_PROPS,
+                    },
+                    {
+                        key: "Sorting",
+                        text: "Sorting",
+                        title: "How to sort the files shown in the file list",
+                        onClick: () => {
+                            props.onCollapse();
+                            dispatch(selection.actions.selectTutorial(SORT_FILES_TUTORIAL));
+                        },
+                        itemProps: OPTION_PROPS,
+                    },
+                    {
+                        key: "Modifying columns in file list",
+                        text: "Modifying columns in file list",
+                        title: "How to modify the columns present in the file list",
+                        onClick: () => {
+                            props.onCollapse();
+                            dispatch(selection.actions.selectTutorial(MODIFY_COLUMNS_TUTORIAL));
+                        },
+                        itemProps: OPTION_PROPS,
+                    },
+                    {
+                        key: "Opening files in another application",
+                        text: "Opening files in another application",
+                        title:
+                            "How to open a file in another application without downloading or copying and pasting the file path",
+                        onClick: () => {
+                            props.onCollapse();
+                            dispatch(selection.actions.selectTutorial(OPEN_FILES_TUTORIAL));
+                        },
+                        itemProps: OPTION_PROPS,
+                    },
+                    {
+                        key: 'Creating "Collections"',
+                        text: 'Creating "Collections"',
+                        title:
+                            'How to create a "Collection" of files for preservation, ML, or sharing purposes',
```

Comment:
nit Should these titles attrs end in `.` ? it looks like they are consistently not, which seems fine as long as it is intentional, and consistent.